src/backend/postgres/types.rs

```diff
@@ -35,9 +35,13 @@ impl TypeBuilder for PostgresQueryBuilder {
             write!(sql, "IF EXISTS ").unwrap();
         }
 
-        for name in drop.names.iter() {
```

Comment:
`iter().enumerate()` and `if i > 0` might align more with the coding style of the repo.